sdk/maps/azure-maps-elevation/src/main/java/com/azure/maps/elevation/implementation/helpers/Utility.java

```diff
@@ -0,0 +1,32 @@
+package com.azure.maps.elevation.implementation.helpers;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import com.azure.core.models.GeoBoundingBox;
+import com.azure.core.models.GeoPosition;
+import com.azure.maps.elevation.implementation.models.LatLongPairAbbreviated;
+
+public class Utility {
+    
+    public static List<LatLongPairAbbreviated> toLatLongPairAbbreviated(List<GeoPosition> points) {
+        List<LatLongPairAbbreviated> points2 = new ArrayList<>();
+        for (GeoPosition point : points) {
+            points2.add(new LatLongPairAbbreviated().setLat(point.getLatitude()).setLon(point.getLongitude()));
+        }
+        return points2;
+    }
+
+    public static List<String> geoPositionToString(List<GeoPosition> points) {
+        List<String> points2 = new ArrayList<>();
+        for (GeoPosition point : points) {
+            points2.add(point.getLongitude() + "," + point.getLatitude());
+        }
+        return points2;
+    }
+
+    public static List<Float> geoBoundingBoxToFloat(GeoBoundingBox boundingBox) {
```

Comment:
you may want to re-check this in the swagger. coordinates should be represented as double, not float. it's not a terribly big deal but we need to be consistent.

Reply:
It is currently format: float in swagger, should I introduce and push changes in swagger for 1. changing float to double and 2. deleting the 2 lines of "x-ms-skip-url-encoding": true?